packages/itinerary-body/src/styled.tsx

```diff
@@ -571,15 +558,22 @@ export const StepDescriptionContainer = styled.div`
 `;
 
 export const StepsHeaderAndMapLink = styled.span`
-  display: inline-block;
+  display: inline-flex;
   align-self: center;
   margin-top: 10px;
+
+  a {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
What is this change for? It squeezes the map button in the screenshot and makes spacing inconsistent with the map button in the expanded steps.

<img width="266" alt="image" src="https://user-images.githubusercontent.com/56846598/228886479-09f7588f-c945-40cd-ade3-50824e157564.png">

Reply:
I agree the new placement has some issues

Reply:
Ok, please add a comment that this accommodates Firefox and Safari.

Reply:
Also, for the mapillary button, could you change `padding-left` to 4 or 5px?